src/polyfem/assembler/Laplacian.cpp

```diff
@@ -15,6 +15,7 @@ namespace polyfem::assembler
 		const Eigen::MatrixXd &gradj = data.vals.basis_values[data.j].grad_t_m;
 		// return ((gradi.array() * gradj.array()).rowwise().sum().array() * da.array()).colwise().sum();
 		double res = 0;
+		// why not just loop over quadrature points here?
```

Comment:
it is the same, maybe you can add an assert to make it explicit

Reply:
Sure, I can add one.